src/backer/backering.py

```diff
@@ -383,6 +388,7 @@ def __init__(self, mbx, pre, topics, retry=5000, hab=None, ledger=None):
         self.retry = retry
         self.hab = hab
         self.ledger = ledger
+        self.queue = queueing.Queueing(hab=hab)
```

Comment:
I would expect we want one instance of `queue` - so pass down the one from the start. And remove the `ledger` field if its not needed. We make sure `queue` is always passed and if it is, we can queue it